### ISSUE TITLE
How to include images in a tweet

### ISSUE THREAD

**Issue by iamakimmer** [tkporter/react-native-fabric-twitterkit#28]
Is it possible to include an image in a tweet using this library?

**Comment by taylorgoolsby** [comment]
No, it is possible to do this using twitter kit, but this library does not expose that feature through react-native. If you know obj-c, it should be fairly easy to expose this feature.
